src/java/org/apache/cassandra/service/accord/fastpath/ParameterizedFastPathStrategy.java

```diff
@@ -146,7 +146,7 @@ static WeightedDc fromString(String s, int idx)
             else if (parts.length == 2)
                 return new WeightedDc(validateDC(parts[0]), validateWeight(parts[1]), false);
             else
-                throw cfe("Invalid dc weighting syntax %s, use <dc>:<weight>");
+                throw cfe("Invalid dc weighting syntax %s, use <dc>:<weight>", s);
```

Comment:
this was the error we saw in UI, and its because `foo:bar:42`... the `dc` had `:` in the name, so hit this logic, but since `, s` was missing it threw an issue with string formatting rather than the CFE we expected